skimage/segmentation/_felzenszwalb_cy.pyx

```diff
@@ -12,50 +12,54 @@ from ..measure._ccomp cimport find_root, join_trees
 from ..util import img_as_float
 
 
-def _felzenszwalb_grey(image, double scale=1, sigma=0.8,
+def _felzenszwalb_rgb(image, double scale=1, sigma=0.8,
                        Py_ssize_t min_size=20):
-    """Felzenszwalb's efficient graph based segmentation for a single channel.
+    """Felzenszwalb's efficient graph based segmentation for multiple channels.
```

Comment:
Actually, I think here and below in the error message it is more correct to declare that the function accepts both single and multichannel images, but `ndim` has to be 3.